types.go

```diff
@@ -590,11 +590,19 @@ type (
 		UpdateTime    *time.Time             `json:"update_time,omitempty"`
 	}
 
+	// SellerReceivableBreakdown has the detailed breakdown of the capture activity.
+	SellerReceivableBreakdown struct {
```

Comment:
any reason we aren't adding fields like `paypal_fee_in_receivable_currency` etc?